pom.xml

```diff
@@ -229,6 +229,13 @@
                     <compatibleSinceVersion>2.0</compatibleSinceVersion>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <forkCount>0</forkCount>
```

Comment:
what's this needed for?

Reply:
I needed this for the build (using `mvn install`) to finish. I was getting the build error below:
https://del.dog/asinewohif.sql

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.20:test (default-test) on project slack: There are test failures.
...
[ERROR] Error occurred in starting fork, check output in log
[ERROR] Process Exit Code: 1
```

Build system: Windows 10 WSL, debian subsystem, openjdk-8-jdk java SDK

Reply:
I just confirmed this isn't related to this commit, I also need it to build the current master. Maybe I can push it as a different PR @timja, sorry about that

Reply:
please, not keen to disable it globally, should be able to pass it on the command line though, I had a quick google and couldn't see anything

Reply:
Ok, I will remove that part and also rebase to current master before force pushing into this PR to make the commit history cleaner, is that ok with you?

Reply:
sure sounds good, other wise good to merge after you get a green build on the rebase

Reply:
Ok, I found out that it's a bug in `openjdk-8-jdk`: https://stackoverflow.com/a/53016532/4008886
Debian bug: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=911925
[Workaround without involving maven pom.xmls](https://issues.apache.org/jira/browse/SUREFIRE-1588?focusedCommentId=16671777&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16671777)